jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull-e2e.yaml

```diff
@@ -150,6 +150,68 @@
                 fi
                 echo "Exiting with code: ${{rc}}"
                 exit ${{rc}}
+        - 'gci-build-test-e2e-gce': # kubernetes-pull-gci-build-test-e2e-gce
+            cmd: |
+                export KUBE_SKIP_PUSH_GCS=y
+                export KUBE_RUN_FROM_OUTPUT=y
+                export KUBE_FASTBUILD=true
+                ./hack/jenkins/build.sh
+                if [[ "${{ghprbTargetBranch:-}}" == "release-1.0" || "${{ghprbTargetBranch:-}}" == "release-1.1" ]]; then
+                  ./hack/jenkins/e2e.sh
+                else
+                  # Nothing should want Jenkins $HOME
+                  export HOME=${{WORKSPACE}}
+
+                  export KUBERNETES_PROVIDER="gce"
+                  export E2E_MIN_STARTUP_PODS="1"
+                  export KUBE_GCE_ZONE="us-central1-f"
+                  export FAIL_ON_GCP_RESOURCE_LEAK="true"
+
+                  # Flake detection. Individual tests get a second chance to pass.
+                  export GINKGO_TOLERATE_FLAKES="y"
+
+                  export E2E_NAME="e2e-gce-${{NODE_NAME}}-${{EXECUTOR_NUMBER}}"
+                  export GINKGO_PARALLEL="y"
+                  # This list should match the list in kubernetes-e2e-gce.
+                  export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+                  export FAIL_ON_GCP_RESOURCE_LEAK="false"
+                  export PROJECT="k8s-jkns-pr-gci-gce"
```

Comment:
I had created the project, but you're right, it needs a quota bump to match `k8s-jkns-pr-gce`.